src/PileupTrack.js

```diff
@@ -188,9 +187,14 @@ const PileupTrack = (HGC, ...args) => {
 
   class PileupTrackClass extends HGC.tracks.Tiled1DPixiTrack {
     constructor(context, options) {
+      let baseUrl = `${getThisScriptLocation()}/`;
+      if (options.workerScriptLocation) {
```

Comment:
Could you add this option to the README?

Reply:
Done